reinforcement_learning/rlclientlib/model_mgmt/model_mgmt.cc

```diff
@@ -40,8 +40,10 @@ namespace reinforcement_learning {
     }
 
     void model_data::free() {
-      delete[] _data;
-      _data = nullptr;
+      if (_data != nullptr) {
```

Comment:
was this causing an issue in windows test?

Reply:
Potentially. Jacob made this change. The behavior I was seeing that the returned `model_data` object from `get_model_data_from_raw` contained garbage memory despite being copied to right before. This feels like a compiler bug to me because not only should that object have been valid, it should have also followed copy elision and so not have to be copied and deleted. Ah well, this is a simple mitigation.

Reply:
I was seeing a free on nullptr here. It might no longer be an issue with the real fix.

Reply:
It doesn't hurt to do the check. We don't want to double free